ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -249,14 +267,12 @@ export default defineComponent({
       areAllSelected: computed((): boolean => {
         return localState.results?.every(result => result && result.selected === true)
       }),
-      isIndeterminate: computed((): boolean => {
+      isIndeterminate: computed(() => {
```

Comment:
Why remove the return type?
Also, i would remove any code that isn't used, even if there is future consideration for implementation.

Reply:
cuz I removed it and added it back but forgot to add the return type,
now I'm deleting it again haha

Reply:
😆 Sounds good, thanks sir!